### PR TITLE
WELD-2366 Review Weld 3 docs - I. Beans.

### PR DESCRIPTION
**Pull request by manovotn**
Most are minor changes.
I removed the sections about `@New` qualifier, as with Weld 3.x, people should know better :)

### CODE REVIEW

File: docs/reference/src/main/asciidoc/injection.asciidoc [hunk 1]
```diff
@@ -551,6 +558,12 @@ Annotation qualifier = synchronously ?
 PaymentProcessor p = anyPaymentProcessor.select(qualifier).get().process(payment);
 ----------------------------------------------------------------------------------
 
+[NOTE]
```
Comment (tremes) [inline docs/reference/src/main/asciidoc/injection.asciidoc]: I would move this note to `4.4. The built-in qualifiers @Default and @Any` chapter

File: docs/reference/src/main/asciidoc/injection.asciidoc [hunk 2]
```diff
@@ -551,6 +558,12 @@ Annotation qualifier = synchronously ?
...
+====
+Since CDI 2.0, most annotations from `javax.enterprise` package have their `AnnotationLiteral` implementations.
+Therefore, in order to to programatically obtain (for instance) `@Any` annotation, you can simply do `Any.Literal.INSTANCE`.
```
Comment (tremes) [inline docs/reference/src/main/asciidoc/injection.asciidoc]: Typo - double `to`.

Comment (manovotn) [inline docs/reference/src/main/asciidoc/injection.asciidoc]: Thanks

File: docs/reference/src/main/asciidoc/injection.asciidoc (hunk 1 above)
Comment (manovotn) [inline docs/reference/src/main/asciidoc/injection.asciidoc]: Hmm, I disagree. 
In `4.4` we don't say a word about implementing a qualifier in order to use it programmatically.
Here, on the contrary, we do, so I think we should straight away say that you needn't implement all of them.

**Pull request merged**